dto/src/main/java/org/jboss/pnc/dto/response/Vertex.java

```diff
@@ -32,10 +34,19 @@
 @JsonDeserialize(builder = Vertex.Builder.class)
 public class Vertex<T> {
 
+    /**
+     * Name of the object.
+     */
     private final String name;
 
+    /**
+     * The object type.
+     */
     private final String dataType;
 
+    /**
+     * The object stored in the vertex.
+     */
     private final T data;
 
```

Comment:
Extend the javadocs, please.

Reply:
@matejonnet Can you please provide the info I should write there?

Reply:
Current doc seems fine to me.